Automattic/woocommerce-payments
# Replace existing manual scripts asset loading with dependency extraction plugin

Fixes #5359

#### Changes proposed in this Pull Request

To load script dependencies, we used `@woocommerce/dependency-extraction-webpack-plugin` that generates *.asset.php file with the script dependencies from WP and WC. In some parts of code, we are still using the "traditional" or "default" way of loading scripts. This PR address this issue by replacing these parts of code with assets loading provided by the NPM plugin mentioned above.

#### Testing instructions

Make sure that all scripts are loaded and make sure that E2E tests and JS tests are loading fine.

*

-------------------


**Post merge**
